Neos.Fusion.Form/Classes/FusionObjects/FieldComponentImplementation.php

```diff
@@ -0,0 +1,166 @@
+<?php
+declare(strict_types=1);
+
+namespace Neos\Fusion\Form\FusionObjects;
+
+/*
+ * This file is part of the Neos.Fusion.Form package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Fusion\Exception;
+use Neos\Fusion\FusionObjects\ComponentImplementation;
+use Neos\Fusion\Form\Domain\Model\FormDefinition;
+use Neos\Fusion\Form\Domain\Model\FieldDefinition;
+use Neos\Utility\ObjectAccess;
+
+/**
+ * Class FormFieldImplementation
+ * @package Neos\Fusion\Form\FusionObjects
+ */
+class FieldComponentImplementation extends ComponentImplementation
+{
+    /**
+     * Properties that are ignored and not included into the ``props`` context
+     *
+     * @var array
+     */
+    protected $ignoreProperties = ['__meta', 'renderer', 'content'];
+
+    /**
+     * @var FormDefinition
+     */
+    protected $form;
+
+    /**
+     * @param array $context
+     * @return array
+     */
+    protected function prepare($context): array
+    {
+        if (array_key_exists('form', $context) && $context['form'] instanceof FormDefinition) {
+            $this->form = $context['form'];
+        }
+
+        $field = new FieldDefinition(
+            $this->getName(),
+            $this->getValue(),
+            $this->getMultiple()
+        );
+
+        $context['field'] = $field;
+
+        // make sure the field is already in the context once content is evaluated
+        $this->runtime->pushContextArray($context);
+        $content = $this->fusionValue('content');
```

Comment:
I'd use $this->runtime->render() here, as it makes it explicit whan the content is evaluated.